README.md

```diff
@@ -245,8 +245,8 @@ The best way to debug in batch mode is to provide `--output` file. Returned data
 
 
 
-## Assembling a training model code folder <a name="training_model_folder"></a>
-Custom training models are in active development. They include a `fit()` function, can be trained on the Leaderboard, benchmarked against DataRobot AutoML models, and get access to DataRobot's full set of automated insights. Refer to the [quickrun readme](QUICKSTART-FOR-TRAINING.md).
+## Assembling a custom task code folder <a name="training_model_folder"></a>
+Custom tasks are in active development. They include a `fit()` function, can be trained on the Leaderboard, benchmarked against DataRobot AutoML models, and get access to DataRobot's full set of automated insights. Refer to the [quickrun readme](QUICKSTART-FOR-TASKS.md).
```

Comment:
Could a `custom task` be a transformer? Because only `estimator` may include the `fit` method. If no, consider using explicitly the `estimator` word. This comment also applies to other references in this PR.

Reply:
fit is required for all custom tasks.